### PR TITLE
Added email validation for CC and BCC field available in the email body

### PR DESCRIPTION
**Pull request by Sachinbisht27**
Targets #2174 

### Description
Added email validation for CC and BCC field available in the invoice reminder email body.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Invoice/Resources/views/modals/invoice-reminder.blade.php [hunk 1]
```diff
@@ -43,7 +43,7 @@ class="form-control" required>
                                     <i class="fa fa-question-circle"></i>
                                 </span>
                             </label>
-                            <input type="text" name="bcc" id="bcc" class="form-control" value="">
+                            <input type="email" name="bcc" id="bcc" class="form-control" value="{{ config('invoice.mail.send-invoice.email') }}">
```
Comment (gautam071) [inline Modules/Invoice/Resources/views/modals/invoice-reminder.blade.php:46]: Already via CC, the mail is sent to the same email we don't need this,
we can remove this line 
```suggestion
                            <input type="email" name="bcc" id="bcc" class="form-control" value="">
```

**Review by gautam071 — COMMENTED** [review]
One suggestion

File: Modules/Invoice/Resources/views/modals/invoice-reminder.blade.php (hunk 1 above)
Comment (Sachinbisht27) [reply Modules/Invoice/Resources/views/modals/invoice-reminder.blade.php:46]: Yes, sure getting the point.

**Review by gautam071 — APPROVED** [review]
Looks good

**Review by gautam071 — CHANGES_REQUESTED** [review]
looks good!

**Review by gautam071 — APPROVED** [review]

**Pull request merged**